### PR TITLE
Create a notification (Backend only)

### PR DESCRIPTION
**Pull request by vando2108**

### CODE REVIEW

File: judge/views/contests.py [hunk 1]
```diff
@@ -1023,3 +1023,59 @@ def post(self, request, *args, **kwargs):
...
+
+    def get_form_kwargs(self):
+        kwargs = super(CreateNotification, self).get_form_kwargs()
+        # Due to some limitation with query set in select2
+        # We only support this if the contest is private for only
+        # 1 organization
+        if self.object.organizations.count() == 1:
+            kwargs['org_pk'] = self.object.organizations.values_list('pk', flat=True)[0]
+
+        return kwargs
```
Comment (leduythuccs) [inline judge/views/contests.py:1051]: this method is useless in this view.

File: judge/views/problem.py [hunk 2]
```diff
@@ -796,6 +797,7 @@ def get_content_title(self):
     def form_valid(self, form):
         self.object = problem = form.save()
         problem.suggester = self.request.user.profile
+        problem.allowed_languages.set(Language.objects.all())
```
Comment (leduythuccs) [inline judge/views/problem.py:800]: remove

**Review by leduythuccs — CHANGES_REQUESTED** [review]

File: judge/forms.py [hunk 3]
```diff
@@ -554,3 +554,14 @@ class Meta:
...
+
+class CreateNotificationForm(ModelForm):
+    def __init__(self, *args, **kwargs):
+        org_pk = kwargs.pop('org_pk', None)
+        super(CreateNotificationForm, self).__init__(*args, **kwargs)
+
+    class Meta:
+        model = Contest
+        fields = [
+           'description',
```
[the remark this answers is not in the document]
Comment (vando2108) [reply judge/forms.py:560]: I reuse this Form for create a notification

File: judge/views/contests.py (hunk 1 above)
Comment (vando2108) [reply judge/views/contests.py:1051]: my bad :< I was remove this

File: judge/forms.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (leduythuccs) [reply judge/forms.py:560]: I saw the `CreateNotificationCustomForm`, so why this you use this?

**Comment by leduythuccs** [comment]
Close due to #171

**Pull request closed without merging**